triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/RunnerConfig.scala

```diff
@@ -211,6 +214,20 @@ object RunnerConfig {
       }
       .text(s"Application ID used to submit commands. Defaults to ${DefaultApplicationId}")
 
+    opt[Unit]('v', "verbose")
+      .text("Root logging level -> DEBUG")
+      .action((_, cli) => cli.copy(rootLoggingLevel = Some(Level.DEBUG)))
+
+    opt[Unit]("debug")
+      .text("Root logging level -> DEBUG")
+      .action((_, cli) => cli.copy(rootLoggingLevel = Some(Level.DEBUG)))
+
+    implicit val levelRead: scopt.Read[Level] = scopt.Read.reads(Level.valueOf)
+    opt[Level]("log-level-root")
+      .text("Log-level of the root logger")
+      .valueName("<LEVEL>")
+      .action((level, cli) => cli.copy(rootLoggingLevel = Some(level)))
```

Comment:
If anyone wants to include any of the other Canton CLI logging options, then please chime in.